pkg/gardenlet/operation/botanist/kubeapiserver.go

```diff
@@ -156,7 +155,7 @@ func (b *Botanist) computeKubeAPIServerSNIConfig() kubeapiserver.SNIConfig {
 }
 
 // DeployKubeAPIServer deploys the Kubernetes API server.
-func (b *Botanist) DeployKubeAPIServer(ctx context.Context) error {
+func (b *Botanist) DeployKubeAPIServer(ctx context.Context, enableNodeAgentAuthorizer bool) error {
```

Comment:
Could you enhance the [unit tests](https://github.com/gardener/gardener/blob/master/pkg/gardenlet/operation/botanist/kubeapiserver_test.go) for this case?